cms/envs/common.py

```diff
@@ -652,7 +652,7 @@
 REQUIRE_BUILD_PROFILE = "cms/js/build.js"
 
 # The name of the require.js script used by your project, relative to REQUIRE_BASE_URL.
-REQUIRE_JS = "js/vendor/require.js"
+REQUIRE_JS = "js/vendor/requiresjs/require.js"
```

Comment:
It already held the RequireJS.text library, so I liked the idea of keeping them together.